.github/ISSUE_TEMPLATE/regression.yaml

```
@@ -55,6 +55,4 @@ body:
         Thanks for submitting this issue! Are you a contributor? If not, have you thought about it? 
 
         Argo Workflows is seeking more community involvement and ultimately more [Reviewers and Approvers](https://github.com/argoproj/argoproj/blob/main/community/membership.md) to help keep it viable. 
-        We are gauging interest in a potential system in which many companies pledge a little bit of time each to help get more people into these roles. 
-        See [#12229](https://github.com/argoproj/argo-workflows/issues/12229) for more information. 
-        If you think you or your company may be interested in getting involved, please add a comment to the issue.
+        See [here](https://github.com/argoproj/argo-workflows/blob/main/community/sustainability_effort.md) for more information. 
```

```suggestion
        See [Sustainability Effort](https://github.com/argoproj/argo-workflows/blob/main/community/sustainability_effort.md) for more information. 
```

- Avoid ambiguous "here", [per k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#links)